definitions/procedures/backup/config_files.rb

```diff
@@ -45,10 +45,24 @@ def run
         end
       end
     end
-    # rubocop:enable Metrics/MethodLength
 
-    # rubocop:disable Metrics/AbcSize
     def config_files
+      all_configs = available_features_config
+      configs = all_configs.first
+      exclude_configs = all_configs.last
+
+      if feature(:foreman_proxy)
+        configs += foreman_proxy_configs
+        exclude_configs += foreman_proxy_exclude_configs
+      end
+
+      configs += deb_config_dirs if debian_or_ubuntu?
```

Comment:
A bit of a nit, but there is a bit of duplication going on here. I think I'd name it `config_dirs` and have all the `debian_or_ubuntu?` logic there. If it isn't, return an empty array. Then this becomes:
```suggestion
      configs += config_dirs
```